app/src/androidTest/kotlin/com/k0d4black/theforce/TheForceTestApplication.kt

```diff
@@ -1,18 +1,29 @@
 package com.k0d4black.theforce
 
-import com.k0d4black.theforce.di.AppComponent
-import com.k0d4black.theforce.di.DaggerTestApplicationComponent
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.startKoin
+import org.koin.core.context.unloadKoinModules
+import org.koin.core.module.Module
 
 class TheForceTestApplication : TheForceApplication() {
```

Comment:
To start fixing the android UI test, the test application instance needs re-configuration.
Koin needs to be launched once hence a Koin context already exists error will be thrown.
To do this:

- Inherit from the ```Application``` class instead , that solves the parent class ```TheForceApplication``` from starting up  Koin.
```kotlin
class TheForceTestApplication : Application()
```